src/lib/reasoners/fun_sat.ml

```diff
@@ -1119,12 +1119,12 @@ module Make (Th : Theory.S) = struct
       if not (Options.get_first_interpretation ()) then compute
       else !(env.last_saved_model) == None
     in
-    if not compute then env
+    if not compute || not @@ Options.get_interpretation () then env
```

Comment:
I believe that `compute` can only be true if `Options.get_interpretation ()` is true, and so this check is meaningless.